app/DeepResearch.Web/appsettings.Development.json

```
@@ -6,10 +6,11 @@
     }
   },
   "OpenAI": {
-    "Endpoint": "https://swcen-aoai.openai.azure.com/",
+    "Endpoint": "https://<<Your AI Foundry name>>.cognitiveservices.azure.com",
```

```suggestion
    "Endpoint": "https://<<Your AI Foundry name>>.azure.com",
```
cognitiveservicesは不要？

---

AOAI Service ではなく Azure AI Foundry でモデルデプロイしたときのエンドポイントがこんな感じだったはず＞

---

ミスリーディングになりそうなので両方対応できるようにazure.comだけにしておきます。